tests/integration/models/get_enabled_features_params.go

```diff
@@ -16,8 +16,8 @@
 
 package models
 
-// GetEnabledFeaturesParams represents params required for GetEnabledFeatures API
-type GetEnabledFeaturesParams struct {
+// GetEnabledFeaturesRequestParams represents params required for GetEnabledFeatures API
+type GetEnabledFeaturesRequestParams struct {
```

Comment:
why you changed names?

Reply:
to follow the same naming scheme for all request params.